http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -265,13 +266,13 @@ class LeonardoServiceSpec
   it should "create a cluster with a client-supplied welder image" in isolatedDbTest {
     implicit val patienceConfig = PatienceConfig(timeout = 1.second)
 
-    val customWelderImage = Some("my-custom-welder-image-link")
+    val customWelderImage = GCR("my-custom-welder-image-link")
 
     // create the cluster
     val clusterRequest = testClusterRequest.copy(
       machineConfig = Some(singleNodeDefaultMachineConfig),
       stopAfterCreation = Some(true),
-      welderDockerImage = customWelderImage,
+      welderDockerImage = Some(customWelderImage),
       enableWelder = Some(true)
```

Comment:
Since the [corresponding logic](https://github.com/DataBiosphere/leonardo/blob/be64e89cb2637796039d22953fb2945e66396e5c/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala#L1379) is a bit complicated, I wonder if we should test the client-supplied Welder image case with `enableWelder = Some(false)` or if it is too obscure to care about.